webwolf/pom.xml

```diff
@@ -66,6 +66,16 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
+            <exclusions>
```

Comment:
Excluding log4j api is not necessary (the attack is only when you include log4j-core)